curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-048.md

```diff
@@ -0,0 +1,189 @@
+---
+id: 60b69a66b6ddb80858c515a5
+title: Part 48
+challengeType: 0
+dashedName: part-48
+---
+
+# --description--
+
+Set the `z-index` of the `white-paper` to `1`.
```

Comment:
Takes up to this lesson to see a visual change in what z-index does. Not a train-smash - just mentioning it.